### PR TITLE
HPCC-24838 Hide Disk Usage when Containerized

### PR DESCRIPTION
**Pull request by GordonSmith**
Disabled landing zones
Disabled despray

Signed-off-by: Gordon Smith <GordonJSmith@gmail.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [x] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

**Review by jbrundage — DISMISSED** [review]

**Comment by GordonSmith** [comment]
@richardkchapman ready to be pulled

File: esp/src/eclwatch/DFUQueryWidget.js [hunk 1]
```diff
@@ -794,8 +794,7 @@ define([
             registry.byId(this.id + "CopyDropDown").set("disabled", !hasSelection);
             registry.byId(this.id + "RenameDropDown").set("disabled", !hasSelection);
             registry.byId(this.id + "AddtoDropDown").set("disabled", !hasSelection);
-            registry.byId(this.id + "AddtoDropDown").set("disabled", !hasSelection);
-            registry.byId(this.id + "DesprayDropDown").set("disabled", !hasSelection);
+            registry.byId(this.id + "DesprayDropDown").set("disabled", !hasSelection || dojoConfig.isContainer);
```
Comment (ghalliday) [inline esp/src/eclwatch/DFUQueryWidget.js:797]: Why is this disabled for container?  I would have expected exporting/despraying to still be relevant.

File: esp/src/eclwatch/templates/HPCCPlatformFilesWidget.html [hunk 2]
```diff
@@ -6,12 +6,12 @@
         <div id="${id}TabContainer" data-dojo-props="region: 'center', tabPosition: 'top'" style="width: 100%; height: 100%" data-dojo-type="dijit.layout.StackContainer">
             <div id="${id}_LogicalFiles" title="${i18n.LogicalFiles}" data-dojo-props="delayWidget: 'DFUQueryWidget'" data-dojo-type="DelayLoadWidget">
             </div>
-            <div id="${id}_LandingZones" title="${i18n.LandingZones}" data-dojo-props="delayWidget: 'LZBrowseWidget'" data-dojo-type="DelayLoadWidget">
+            <div id="${id}_LandingZones" title="${i18n.LandingZones}" data-dojo-props="delayWidget: 'LZBrowseWidget', disabled:${dojoConfig.isContainer}" data-dojo-type="DelayLoadWidget">
```
Comment (ghalliday) [inline esp/src/eclwatch/templates/HPCCPlatformFilesWidget.html:9]: Is this browsing files on a landing zone?  I think this still makes sense (although isn't currently working for azureblobs).

**Review by ghalliday — COMMENTED** [review]
@GordonSmith a couple of questions

File: esp/src/eclwatch/DFUQueryWidget.js (hunk 1 above)
Comment (GordonSmith) [reply esp/src/eclwatch/DFUQueryWidget.js:797]: I can leave it enabled if the expectation is that we will have Azure landing zone support for v8?

File: esp/src/eclwatch/templates/HPCCPlatformFilesWidget.html (hunk 2 above)
Comment (GordonSmith) [reply esp/src/eclwatch/templates/HPCCPlatformFilesWidget.html:9]: I can leave it enabled if the expectation is that we will have Azure landing zone support for v8?

Comment (ghalliday) [reply esp/src/eclwatch/templates/HPCCPlatformFilesWidget.html:9]: I am planning to implement it in the next few weeks.

File: esp/src/eclwatch/DFUQueryWidget.js (hunk 1 above)
Comment (ghalliday) [reply esp/src/eclwatch/DFUQueryWidget.js:797]: This should be working when Shamser's PR is merged.

**Comment by GordonSmith** [comment]
@ghalliday those changes have been reverted as discussed.

**Review by ghalliday — APPROVED** [review]

**Pull request merged**